packages/cms-lib/errorHandlers.js

```diff
@@ -181,7 +183,7 @@ function logApiStatusCodeError(error, context) {
     const request = context.request
       ? `${action} ${preposition} "${context.request}"`
       : action;
-    messageDetail = `${request} on portal ${context.portalId}`;
+    messageDetail = `${request} in portal ${context.portalId}`;
```

Comment:
Didn't someone (@williamspiro @gcorne) propose we start subbing *"account"* for *"portal"*

Reply:
Yes, I believe that is the plan, but not _just yet_ AFAIK.

Reply:
Yeah, I think we should tackle that separately. This reminded me to file #68.